VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -160,38 +200,75 @@ final public class AuthenticationController
     
     // MARK: - Private Authentication
     
+    /**
+     **(PRIVATE: Vimeo Use Only)**
+     Log in with an email and password
+     
+     - parameter email:      a user's email
+     - parameter password:   a user's password
+     - parameter completion: handler for authentication success or failure
+     */
```

Comment:
I'll augment the docs to be more clear about it not working for third parties.  I'll stop short of adding an actual warning, though, because that'd clutter our own projects :/